### PR TITLE
Typo "relationship**.**"→"relationship **.**"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/sql/reporting-services/report-data/relational-query-designer-user-interface-report-builder?view=sql-server-ver15
#PingMSFTDoc

### CODE REVIEW

**Comment by PRMerger19** [comment]
@hyoshioka0128 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reporting-services/report-data/relational-query-designer-user-interface-report-builder.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ ms.author: maggies
   
 -   **Edit Fields** Opens the **Edit Related Fields** dialog box in which you add and modify relationships between tables. You chose the fields in the right and left tables to join. You can join multiple fields from the left table and right table to specify multiple join conditions in a relationship. The two fields that join the left and right tables do not need to have the same name. The joined fields must have compatible data types.  
   
--   **Delete Relationship**  Deletes the selected relationship**.**  
+-   **Delete Relationship**  Deletes the selected relationship **.**
```
Comment (maggiesMSFT) [inline docs/reporting-services/report-data/relational-query-designer-user-interface-report-builder.md:145]:
```suggestion
-   **Delete Relationship**  Deletes the selected relationship. 

```

**Review by maggiesMSFT — CHANGES_REQUESTED** [review]
Thanks for spotting that, @hyoshioka0128 ! Good eye. In fact we don't need *any* of those asterisks!

**Comment by maggiesMSFT** [comment]
#sign-off  
PR reviewer: Please accept my change and merge. 
Thanks!

**Pull request merged**